.mergify.yml

```diff
@@ -2,10 +2,11 @@ pull_request_rules:
   - name: merge on CI success and one approval
     conditions:
       - "status-success=Travis CI - Pull Request"
-      - "#approved-reviews-by>=1"
+      - "#approved-reviews-by>=2"
```

Comment:
:question: Why all of the sudden? :O

Reply:
I thought it was the current configuration but it is not indeed.